content/streamlit-cloud/get-started/share-your-app/single-sign-on/index.md

```diff
@@ -5,7 +5,7 @@ slug: /streamlit-cloud/get-started/share-your-app/configuring-single-on-sso
 
 # Configuring Single Sign-on (SSO)
 
-By default all apps deployed with Streamlit Cloud Teams and Enterprise are private, which means that others in your company won't be able to view them unless you give them explicit permission.
+By default all apps deployed from private source code are private to the developers in the workspace, which means that others in your company won't be able to view them unless you give them explicit permission.
```

Comment:
probably should find a better way to say "others in your company", since the free tier expansion of private will now no longer include businesses

Reply:
Will run this concern by STC  and ask them whether the following is more accurate:
> By default all apps deployed from private source code are private to the developers in the workspace. Your apps will not be visible to anyone else unless you give them explicit permission.

Reply:
Pushed a commit that rephrases this paragraph to:
> By default all apps deployed from private source code are private to the developers in the workspace. Your apps will not be visible to anyone else unless you grant them explicit permission.